lib/internal/fs.js

```diff
@@ -4,16 +4,17 @@ const Buffer = require('buffer').Buffer;
 const Writable = require('stream').Writable;
 const fs = require('fs');
 const util = require('util');
-const constants = process.binding('constants').fs;
-
-const O_APPEND = constants.O_APPEND | 0;
```

Comment:
Might want to check if the implicit coercion to a 32-bit value was intentional (e.g. performance related)?